conf/default-colors.yaml

```diff
@@ -0,0 +1,31 @@
+# All color variables will be prefixed automatically
+stylesheets:
```

Comment:
conf inside libraries won't work, they should be kept in the parent module and keep the task libraries simple